docs/shared/angular-standalone-tutorial/2-project-graph.md

```diff
@@ -122,6 +122,7 @@ import { Component, Input } from '@angular/core';
 
 @Component({
   selector: 'store-banner',
+  standalone: true,
```

Comment:
This page isn't linked anywhere.  We should actually delete it.  That's a separate PR though

Reply:
ok no problem